packages/react-components/react-toolbar/src/components/ToolbarRadioButton/useToolbarRadioButton.ts

```diff
@@ -19,7 +19,10 @@ export const useToolbarRadioButton_unstable = (
   const size = useToolbarContext_unstable(ctx => ctx.size);
 
   const { onClick: onClickOriginal } = props;
-  const toggleButtonState = useToggleButton_unstable({ size, checked, ...props }, ref);
+  const toggleButtonState = useToggleButton_unstable(
+    { size, checked, role: 'radio', 'aria-checked': checked, 'aria-pressed': undefined, ...props },
```

Comment:
you need to do something like this:

```tsx
toggleButtonState['aria-pressed] = undefined
```

since the toggle button does not care about user `aria-pressed` value https://github.com/microsoft/fluentui/blob/1eceee82a56f249543daf5ba97934e23cbbfa1ac/packages/react-components/react-button/src/utils/useToggleState.ts#L42